install.sh

```diff
@@ -9,8 +9,9 @@ umask 002
 
 source "$(dirname $0)/install/_lib.sh"  # does a `cd .../install/`, among other things
 
-source dc-detect-version.sh
 source parse-cli.sh
+source dc-detect-version.sh
+source turn-things-off.sh
```

Comment:
`error-handling` should come as early as possible?

Reply:
Fair point. It depends on `$dc` so moving just after that in c48efe096ad614e7860b48f7e6bc5a187d4ecede.

Reply:
Hmm ... thought through this a little further and created 2b9a179270763a2919304fc0337fb2c307a499c0. Eh?